documentation/docs/03-runes/01-state.md

```diff
@@ -68,6 +68,33 @@ Objects and arrays are made deeply reactive by wrapping them with [`Proxies`](ht
 
 > Only POJOs (plain old JavaScript objects) are made deeply reactive. Reactivity will stop at class boundaries and leave those alone
 
+## `$state.link`
+
+Linked state stays up to date with its input:
+
+```js
+let a = $state(0);
+let b = $state.link(a);
+
+a = 1;
+console.log(a, b); // 1, 1
+```
+
+You can temporarily _unlink_ state. It will be _relinked_ when the input value changes:
```

Comment:
I find the meaning here hard to comprehend right away. Would it be simpler to understand something like this: 
You can locally override the value, but the input value change will cause syncing the linked value with the input one?